src/Bluejay.asm

```diff
@@ -174,6 +166,7 @@ DEFAULT_PGM_STARTUP_POWER_MAX EQU 25    ; 0..255 => (1000..2000 Throttle): Maxim
 DEFAULT_PGM_BRAKING_STRENGTH EQU 255    ; 0..255 => 0..100 % Braking
 
 DEFAULT_PGM_SAFETY_ARM EQU 0            ; EDT safety arm is disabled by default
+DEFAULT_48to24_THRESHOLD EQU 170        ; About 66% threshold to change between 48 and 24khz
```

Comment:
Yes, that is what I told you initially, but previous releases use 24, 48, and 96 values for the corresponding pwm modes, so changed my mind to reduce changes on the configuration tool. This is because I choose 24, 48 or any other for 24khz, 48khz and dynamyc pwm modes

Reply:
Yes, makes sense - typed out a comment but did not comment and just resolved the conversation :-) I think having the explicit setting now again makes it obvious anyway.